data/crud/unified/bulkWrite-update-hint-clientError.json

```diff
@@ -281,4 +281,4 @@
       ]
     }
   ]
-}
+}
```

Comment:
Suggest reverting whitespace changes to the spec test files, as it is not relevant to the PR. Can do with:

```bash
# Revert state of files to `master` branch:
git checkout master -- data/crud/unified
```